src/settings/plantSettingsActions.ts

```diff
@@ -0,0 +1,56 @@
+import { readState } from '../state/globalActions';
+import { getCoreContext } from '../state/globalState';
+import { PlantSettings } from '../types/settings';
+import { setSetting } from './globalSettingsActions';
+
+/**
+ * Used for setting or updating the selected plant.
+ *
+ * @export Function from Echo Core.
+ * @param {PlantSettings} plantSettings selected plant data object.
+ */
+export function setSelectedPlant(plantSettings: PlantSettings): void {
```

Comment:
Som vi diskuterte litt, så bør ikke setSelectedPlant være tilgjengelig for alle. Det er kun plantSelector som kan gjøre dette.

Reply:
jeg må tilgjengeligjør dette for den uansett. er det noe måte en kan bare tilgjengelig gjør for en fil?

Reply:
Ikke som jeg vet, du kan typescript bedre enn meg :P

Reply:
Men dette tyder på at vi kanskje koder "feil vei". At det er plantSelector som bør implementere setSelectedPlant. Da er det kun denne som får lov å gjøre denne action'en.

Reply:
Hvis vi lar setSelectedPlant være tilgjengelig for alle, så har vi null kontroll på hva 3. parts moduler finner på, som fort skaper vanskelige bugs for oss å håndtere.